drip/scheduler/cron_scheduler.py

```diff
@@ -0,0 +1,38 @@
+from django.conf import settings
+from apscheduler.schedulers.background import BackgroundScheduler
+
+from django.core.management import call_command
+
+
+DRIP_SCHEDULE = getattr(
+    settings, 'DRIP_SCHEDULE', False
+)
+DRIP_SCHEDULE_DAY_OF_WEEK = getattr(
+    settings, 'DRIP_SCHEDULE_DAY_OF_WEEK', 0
+)
+DRIP_SCHEDULE_HOUR = getattr(
+    settings, 'DRIP_SCHEDULE_HOUR', 0
+)
+DRIP_SCHEDULE_MINUTE = getattr(
+    settings, 'DRIP_SCHEDULE_MINUTE', 0
+)
+DRIP_SCHEDULE_TZ = getattr(
+    settings, 'TIME_ZONE', 'UTC'
+)
```

Comment:
Usually it's a good practice to have these namespaced to the library.
You can import a dictionary for this
```suggestion
DRIP_SETTINGS = getattr(
    settings, 'DRIP_SETTINGS', {}
)
DRIP_SCHEDULE = DRIP_SETTINGS.get('DRIP_SCHEDULE', False)
...
```